desktop/src/cef/input.rs

```diff
@@ -11,14 +11,24 @@ use super::consts::{MULTICLICK_ALLOWED_TRAVEL, MULTICLICK_TIMEOUT, SCROLL_LINE_H
 
 pub(crate) fn handle_window_event(browser: &Browser, input_state: &mut InputState, event: &WindowEvent) {
 	match event {
-		WindowEvent::PointerMoved { position, .. } => {
+		WindowEvent::PointerMoved { position, .. } | WindowEvent::PointerEntered { position, .. } => {
 			input_state.cursor_move(position);
 
 			let Some(host) = browser.host() else {
 				return;
 			};
 			host.send_mouse_move_event(Some(&input_state.into()), 0);
 		}
+		WindowEvent::PointerLeft { position, .. } => {
+			if let Some(position) = position {
+				input_state.cursor_move(position);
+			}
+
+			let Some(host) = browser.host() else {
+				return;
```

Comment:
Make this a one-liner, please.

Reply:
Is from auto formating

Reply:
If the line becomes long enough, it'll force it back to three lines, but it's usually worth trying to get the short version. Occasionally you'll have a bunch of long lines mixed in with one short line, in which case I usually opt for consistency to let them all be the three-liner, but otherwise prefer attempting to get the short version if nearby consistency isn't a concern.